lib/sprockets/cache.rb

```diff
@@ -67,7 +67,8 @@ def self.default_logger
     # cache - A compatible backend cache store instance.
     def initialize(cache = nil, logger = self.class.default_logger)
       @cache_wrapper = get_cache_wrapper(cache)
-      @fetch_cache   = Cache::MemoryStore.new(1024)
+      fetch_cache_size = cache.respond_to?(:max_size) ? cache.max_size / 1000 : 1024
```

Comment:
Why divide by 1000? Why not just have them input the number that they want?

Also dividing an integer by an integer will give us a bad value  `1/1000 # => 0`

Reply:
i have new patches, need to cleanup and a new pull request will follow, maybe next week.